std/traits.d

```diff
@@ -3791,6 +3792,47 @@ template hasElaborateDestructor(S)
     static assert( hasElaborateDestructor!S7);
 }
 
+/**
+ True if `S` or any type embedded directly in the representation of `S`
+ defines elaborate move semantics. Elaborate move semantics are
```

Comment:
I'm not sure if I like 'elaborate move semantics', but the DIP doesn't propose any terminology. Ideas?